client/blocks/login/social.jsx

```diff
@@ -28,7 +37,22 @@ class SocialLoginForm extends Component {
 		this.props.loginSocialUser( 'google', response.Zi.id_token ).then( () => {
 			this.props.onSuccess();
 		} ).catch( error => {
-			this.props.errorNotice( error );
+			if ( error === 'unknown_user' ) {
+				const { notice } = this.props.infoNotice( this.props.translate( 'Creating your account' ) );
```

Comment:
I think the convention is to use ellipsis to denote that the action is in progress (i.e. `Creating your account...`).